code/go/internal/validator/folder_spec.go

```diff
@@ -168,13 +169,13 @@ func (s *folderSpec) validate(folderPath string) ValidationErrors {
 	return errs
 }
 
-func (s *folderSpec) findItemSpec(folderItemName string) (*folderItemSpec, error) {
+func (s *folderSpec) findItemSpec(packageName string, folderItemName string) (*folderItemSpec, error) {
 	for _, itemSpec := range s.Contents {
 		if itemSpec.Name != "" && itemSpec.Name == folderItemName {
```

Comment:
Should we allow the use of keyword placeholders, e.g. `{PACKAGE}`, in item names as well (not just item patterns)?

Reply:
I suspected that you may ask about it. I decided to postpone it until we've a real use-case. So far, this is the only binding between items, so I vote for leaving it only for patterns.

Reply:
Sounds good to me!